.github/stale.yml

```diff
@@ -1,33 +1,39 @@
-# Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+name: Close stale issues
+on:
+  schedule:
+    - cron: '50 1 * * *'
 
-# Number of days of inactivity before a stale issue is closed
-daysUntilClose: 7
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-issue-stale: 60
+          days-before-pr-stale: -1
+          days-before-issue-close: 7
+          # days-before-pr-close: 10
```

Comment:
```suggestion
```

Maybe remove this comment?

Reply:
I kept some of the PR stuff, in case we want to use it later.